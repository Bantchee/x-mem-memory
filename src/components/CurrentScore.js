const CurrentScore = (props) => {
    return (
        <div
            className="pt-1 pb-1 border-4 w-64 flex justify-center rounded-lg border-yellow-400"
        >
            <span
                className="text-red-500 m-auto"
            >
                Current Score: {props.score}
            </span>
        </div>
    )
}

export default CurrentScore;