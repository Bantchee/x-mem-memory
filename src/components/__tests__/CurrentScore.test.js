import { render } from "@testing-library/react"
import CurrentScore from "../CurrentScore"


describe("CurrentScore Component", () => {
    test("renders component", () => {
        const { container } = render(<CurrentScore />);
        expect(container).toMatchSnapshot();
    });
});