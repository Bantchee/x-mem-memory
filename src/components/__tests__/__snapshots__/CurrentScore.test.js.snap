// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CurrentScore Component renders component 1`] = `
<div>
  <div
    class="pt-1 pb-1 border-4 w-64 flex justify-center rounded-lg border-yellow-400"
  >
    <span
      class="text-red-500 m-auto"
    >
      Current Score: 
    </span>
  </div>
</div>
`;
